src/Filesystem/Adapter/AwsS3CloudFrontAdapter.php

```diff
@@ -58,11 +97,11 @@ public function copy($path, $newpath)
     }
 
     /**
-     * @inheritdoc
+     * {@inheritDoc}
```

Comment:
`@inheritDoc` is ok in this context => docblocks that inherit everything
`{@inheritdoc}` is for docblocks that inherit the summary + description 

The new Cake 4 code sniffer handles this tag in this way